test/Ecdsa2019Cryptosuite.spec.js

```diff
@@ -6,6 +6,21 @@ import {expect} from 'chai';
 import jsigs from 'jsonld-signatures';
 const {purposes: {AssertionProofPurpose}} = jsigs;
 
+import * as EcdsaMultikey from '@digitalbazaar/ecdsa-multikey';
+import {
+  credential,
+  ecdsaMultikeyKeyPair,
+  ecdsaSecp256KeyPair,
+  ecdsaSecp384KeyPair,
+  ecdsaSecp521KeyPair
```

Comment:
So this cryptosuite is only intended to support secp256r1, none of the others -- so we should remove those ... unless there's something I'm missing.

Reply:
Yes, you're right. We wouldn't ever use these keys in this crypto suite. I have removed them.